Minor layout and graphics tweaks

jamesob opened the pull request:
- Add a horizontal logo
- Reposition a few elements on the home page


# Before 
![selection_061](https://user-images.githubusercontent.com/73197/42594568-942c7eca-851d-11e8-8095-46b0a683b0d2.png)

# After
![selection_060](https://user-images.githubusercontent.com/73197/42594576-98442364-851d-11e8-952e-bf9edd68b93a.png)

harding commented on assets/css/main.scss:
```diff
@@ -54,3 +54,23 @@ p.sponsor-quote {
+    margin-top: 2em;
+
+    > .logo {
+        width: 30%;
```
If you put `min-width: 10em;` below the existing `width` line, the graphic will be shown a bit larger on narrow mobiles but will be the same size on the widest displays.

jamesob replied:
Done

harding commented on _layouts/page.html:
```diff
@@ -0,0 +1,18 @@
+---
+# Forked from the default minima layout to include our main.css
```
It already did that on my system, were you having problems?  I don't see any layout-affecting differences when diffing a copy of the `_site` dir generate with this file to one generated without it, nor do I on a visual inspection of the rendered site.

OTOH, even if creating this file now is unnecessary, it's likely something that we'll end up customizing later anyway, so I'm not opposed to this.

harding replied:
Oh, I see, the CSS file was moved from its default location in d091a44f7b12b9ab19668112333c193d84af5a4c and only the homepage layout was updated to reflect that, so without this file, the newsletters don't get style changes.

This is fine.  An alternative is to move the css file back to its original location in /assets/ and drop the special code from the home layout.  I don't think it matters long term if this site is continued to be customized.

jamesob replied:
Yup; I prefer to keep `_layout/page.html` bundled as it's convenient to have the code ready at hand for modification, as well as for easier discovery - had to google around a bit to figure out where the page template lived beforehand.

harding commented on index.md:
```diff
@@ -4,8 +4,10 @@
-{:style="text-align: center"}
-![OpTech Logo](/img/logos/optech.png)
+
+<div class="logo">
```
I don't have any problem with using HTML here (or anywhere, really), but FYI you can apply a CSS class to a block element in Kramdown like this:

    {:.logo}
    ![OpTech Logo](/img/logos/optech.png)

That will create:

    <p class="logo"><img src="/img/logos/optech.png" alt="OpTech Logo" /></p>

Other things you can do:
- `{#custom-id}`
- `{:custom-tag: "custom-value}`
- Plus other fancier stuff, see the [manual](https://kramdown.gettalong.org/syntax.html#block-ials)

jamesob replied:
Oh, neat! Fixed.

harding commented on assets/css/main.scss:
```diff
@@ -54,3 +54,23 @@ p.sponsor-quote {
+        margin-bottom: 2em;
+    }
+
+    div#sponsors {
```
If this vertical whitespace looks good over the top of the Sponsors `<h3>`, you might want to consider applying it to every `<h3>` on the website, both for consistency and for the avoidance of having too many ad hoc rules that may making figuring out style things later harder.

jnewbery commented:
Requires rebase (and addressing harding's comments).

jamesob commented on assets/css/main.scss:
```diff
@@ -54,3 +54,23 @@ p.sponsor-quote {
 }
+
+a.site-title {
+    width: 30%;
```
Good catch, thanks!

jamesob commented:
I've incorporated @harding's good feedback (thanks, Dave) and I've changed the inline logo to remove the text, as it felt like we were repeating "Bitcoin Optech" a bit too much graphically. I can revert if you guys don't like this.

![selection_064](https://user-images.githubusercontent.com/73197/42641396-77c71cc2-85c2-11e8-88ac-2b85c39f4c1a.png)

![selection_065](https://user-images.githubusercontent.com/73197/42641441-8a76a2f2-85c2-11e8-83e1-78ed093bb661.png)

harding commented:
0c258965f7d3795e1771f4aa52006afc0b336147 LGTM.

jnewbery commented:
> The new PNG also has a transparent background

Thanks! That was bugging me :slightly_smiling_face:

jnewbery commented:
0c258965f7d3795e1771f4aa52006afc0b336147 LGTM :rocket:
